src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -49,6 +49,7 @@
         flex: 1 1 auto;
         flex-direction: column;
         width: 100%; // Explicit width required since flex-direction is column
+        min-height: $sidebarContentMinHeight;
```

Comment:
I didn't notice this before but I see now there's two vertical scroll bars that would appear at the same time

multiple scroll bars in the same axis can have some unintuitive behavior to users, is that okay? is that an accessibility concern?

Reply:
@tjuanitas I've reached out to the Product Team and this solution has been accepted. Two scroll bars appear only at higher zoom and this solution lets us avoid impacting the broader user experience. Great attention to the details! Thanks!

Reply:
great, sounds good to me!